gst/nnstreamer/tensor_filter/tensor_filter_custom.c

```diff
@@ -132,9 +132,18 @@ custom_open (const GstTensorFilterProperties * prop, void **private_data)
   g_assert (!ptr->methods->invoke != !ptr->methods->allocate_invoke);   /* XOR! */
 
   if (ptr->methods->allocate_invoke) {
-    NNS_support_custom.allocate_in_invoke = TRUE;
-    NNS_support_custom.destroyNotify = ptr->methods->destroy_notify;
+    filter_framework->allocate_in_invoke = TRUE;
+    filter_framework->destroyNotify = ptr->methods->destroy_notify;
   }
+
+  if (!ptr->methods->getInputDim) {
+    filter_framework->getInputDimension = NULL;
+  } else if (!ptr->methods->getOutputDim) {
```

Comment:
Question: plz check cond to check filter-callback, 'else if'

Reply:
also, plz check multi-custom filters (PR #2029 )

Reply:
Thanks for pointing out the mistake.
I will remove the changes in this PR, and modify it to remove existing errors like the one you pointed out.

Reply:
@jaeyun-jung Updated some of the implementations, and added the required interfaces to update this. Please give an initial review.